src/repo/reservoirRepo.js

```diff
@@ -1,42 +1,29 @@
 import qs from 'qs'
 import { capitalize } from '@/lib/primitive-helpers'
 
-export default function (axios) {
-  return {
-    // Get reservoir by id (fid)
-    getById: id => axios.$get(`reservoir/${id}`),
-
-    // Get only the service water area (raw data)
-    getTimeSeriesById: id =>
-      axios
-        .$get(`reservoir/${id}/ts/surface_water_area`)
-        .then(timeSeries => formatTimeSeries(id, timeSeries)),
-
-    getByGeometry: geometry =>
-      axios.post('reservoir/geometry', geometry).then(({ data }) => data),
-
-    getTimeSeriesByGeometry: (geometry, variableName, period) =>
-      axios.post(`reservoir/geometry/ts/${variableName}?agg_period=${period}`, geometry).then(formatMultipleTimeSeries),
-
-    getByIds: ids => axios.$get(`reservoir/?${qs.stringify({ ids, limit: ids.length }, { indices: false })}`),
-
-    getTimeSeriesByIds: (ids, variableName, period) => {
-      return axios
-        .$get(`ts/?${qs.stringify({
-          reservoir_ids: ids,
-          variable_name: variableName,
-          agg_period: period,
-        }, { indices: false })}`)
-        .then(formatMultipleTimeSeries)
-    },
+const valueConversion = ({ t, value, unit }) => {
+  if (unit === 'm2') {
+    // Convert to km2
+    return [t, (value / 1000000).toFixed(2)]
+  } else if (unit === 'm3') {
+    // Convert to km3
+    return [t, (value / 1000000000).toFixed(2)]
+  } else {
+    return [t, value]
   }
 }
 
-// TODO: make sure this km2 comes from the backend again as an unit
-// const valueUnit = timeSeries[0]?.unit
-const valueUnit = 'km2'
+const unitConversion = (unit) => {
+  if (unit === 'm2') {
+    return 'km2'
+  } else if (unit === 'm3') {
+    return 'km3'
+  } else {
+    return unit
+  }
+}
```

Comment:
Maybe we can combine these 2 in a helper function where we return a label and value so it makes more sense that those belong to each other.

Reply:
There are some concepts of "beautiful" numbers. In d3 for example they have the 's' SI unit notation:
https://github.com/d3/d3-format/tree/v3.1.0